source/create.ts

```diff
@@ -0,0 +1,82 @@
+'use strict';
+import {URL} from 'url';
```

Comment:
Don't import `URL`. We use the `URL` global.

Reply:
@sindresorhus Does TypeScript resolve it automatically if using Node 8?

Reply:
Never mind. I got confused with Ky.